installation/resources/installer.yaml

```diff
@@ -63,6 +63,150 @@ spec:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  name: helm-certs-sa
+  namespace: kyma-installer
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-certs-rolebinding
+  namespace: kube-system
+subjects:
+- kind: ServiceAccount
+  name: helm-certs-sa
+  namespace: kyma-installer
+roleRef:
+  kind: Role
+  name: helm-certs-installer
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-certs-rolebinding
+  namespace: kyma-installer
+subjects:
+- kind: ServiceAccount
+  name: helm-certs-sa
+  namespace: kyma-installer
+roleRef:
+  kind: Role
+  name: helm-certs-installer
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-certs-installer
```

Comment:
Nit: This role is for reading certs from kube-system so I would suggest to change name